src/webots/gui/WbMultimediaStreamingServer.cpp

```diff
@@ -31,11 +31,14 @@
 #include <QtGui/QMouseEvent>
 #include <QtWebSockets/QWebSocket>
 
+#include <iostream>
+
+using namespace std;
+
```

Comment:
Is it really needed?

Reply:
No indeed, that's a remnant of debugging...